src/test/pybind/test_rados.py

```diff
@@ -444,6 +444,13 @@ def test_snap_read(self):
         self.ioctx.remove_snap("snap1")
         self.ioctx.remove_object("inhead")
 
+    def test_assert_exists(self):
+        with WriteOpCtx() as write_op:
+            write_op.new(0)
+            write_op.assert_exists()
+            self.ioctx.operate_write_op(write_op, 'abc')
```

Comment:
could you also test a negative case where the object does not exist? also, please make sure the created object(s) is removed after the test finishes.

Reply:
Yes, I'll test it right away.

Reply:
Updated，please check it，thanks.

Reply:
if there is any problem, please tell me and I will modify it.